src/main/resources/org/jenkinsci/plugins/zap/ZAPDriver/config.jelly

```diff
@@ -128,6 +128,9 @@ SOFTWARE.
             <f:entry title="${%Logged in Indicator}" field="loggedInIndicator">
               <f:textbox clazz="required" />
             </f:entry>
+            <f:entry title="${%Logged out Indicator}" field="loggedOutIndicator">
+              <f:textbox clazz="required" />
```

Comment:
I'd suggest removing the required from both indicators, none of them is really required, also, from a use case mentioned in the mailing list only the logged out indicator would be set.